node/libs/roles/src/proto/validator.proto

```diff
@@ -11,7 +11,7 @@ message Fork {
 
 message Genesis {
   optional Fork fork = 1; // required
-  repeated PublicKey validators = 2;
```

Comment:
We already have a deployment (stage), so please try to keep backward compatibility. It also means that we should support the old way of specifying validators (and make sure that hashes are stable) for the time being.

Reply:
Ok, fixed this creating the 3rd field but tried to keep the `validators` name for it to use `genesis.validators` all around the code, but the checker complains with:
```
Error: Field "2" with name "validator_keys" on message "Genesis" changed option "json_name" from "validators" to "validatorKeys".
Error: Field "2" on message "Genesis" changed name from "validators" to "validator_keys".
Error: buf found 2 breaking changes.
```

So, how should I fix this? 
- Should I change the name to the 3rd field? (that will make the code to use something like `genesis.weighted_validators` everywhere)
- Should I use `[json_name="validators"];` and `[json_name="weighted_validators"];` for 2nd and 3rd fields to let it keep the json names and allow us to use `genesis.validators` in our code? (that will make json representation a bit confusing, not sure if we are using that. Also I haven't tried this, not sure if it will work)
- Should I keep the `validator` name in the `proto::Genesis` struct but not in the `validator::Genesis` one? (that way field names won't match but the code will still be `genesis.validators` everywhere

It's a kind of a silly question, but I'm not comfortable with none of the options here 🤔

Reply:
cool, thanks